nilearn/_utils/tests/test_path_finding.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+
+import pytest
+
+from nilearn._utils.path_finding import resolve_globbing
+
+
+def test_resolve_globbing(tmp_path):
+    assert resolve_globbing(tmp_path) == [tmp_path]
```

Comment:
```suggestion
    assert resolve_globbing(tmp_path) == [tmp_path]
    assert resolve_globbing([]) == []
```

Reply:
I would put the same type of test here, to check the case when path is neither `str` nor `path`.

The doctring can be updated as well as in the case of `stringify_path`. Or alternatively this method could accept only `str` or `Path` and raise error otherwise.